temporian/core/data/duration.py

```diff
@@ -39,6 +59,33 @@ def seconds(value: Union[int, float]) -> Duration:
     this method does nothing else than casting the input to `float`. It may be
     used in order to make the code more explicit.
 
+    Example:
+        ```python
+        >>> # Only converts to float
+        >>> tp.duration.seconds(3)
+        3.0
+
+        ```
+
+    Explicit time units:
+        ```python
+        >>> evset = tp.event_set(timestamps=[1, 2, 6], features={"f1": [1, 5, -5]})
+        >>> source = evset.node()
+
+        >>> # More explicit
+        >>> result = tp.moving_sum(source, window_length=tp.duration.seconds(3))
+
+        >>> # Actually equivalent
+        >>> result = tp.moving_sum(source, window_length=3)
+
+        >>> result.evaluate({source: evset})
+        indexes: ...
+                timestamps: [1. 2. 6.]
+                'f1': [ 1 6 -5]
+        ...
+
+        ```

```

Comment:
I would always use the same small example in all duration functions: print a duration and use a duration in a simple graph.

I would always use datetime timestamps.